src/SDKs/IotHub/IotHub.Tests/IotHub.Tests.csproj

```diff
@@ -4,13 +4,17 @@
     <PackageId>IotHub.Tests</PackageId>
     <Description>IotHub.Tests Class Library</Description>
     <AssemblyName>IotHub.Tests</AssemblyName>
-    <VersionPrefix>1.0.1-preview</VersionPrefix>    
+    <VersionPrefix>1.0.2-preview</VersionPrefix>    
   </PropertyGroup>
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+
```

Comment:
@anusapan please use latest version of Resource manager to record all your tests. You are still using 1.1.0, latest version is 1.6.0
YOU will have add reference to the new nuget, use the namespace from the new nuget and remove all traces of old namespace and then record your tests.

Reply:
added new references of ResourceManager and recorded test again